Sources/ZetaPushNetwork/Client/ClientHelperDelegate.swift

```diff
@@ -11,25 +11,27 @@ import Foundation
 
 // MARK: CometdClientDelegate
 public protocol ClientHelperDelegate: class {
+  func onConnectionFailed(_ client: ClientHelper, error: ClientHelperError)
   func onConnectionEstablished(_ client: ClientHelper)
-  func onConnectionBroken(_ client: ClientHelper)
-  func onConnectionClosed(_ client: ClientHelper)
+  func onConnectionBroken(_ client: ClientHelper, error: ClientHelperError)
+  func onConnectionClosed(_ client: ClientHelper, error: ClientHelperError?)
   func onConnectionClosedAdviceReconnect(_ client: ClientHelper)
   func onSuccessfulHandshake(_ client: ClientHelper)
-  func onFailedHandshake(_ client: ClientHelper)
+  func onFailedHandshake(_ client: ClientHelper, error: ClientHelperError)
```

Comment:
Here you put a ClientHelperError but for the API is it better compare to just Error ? I don't know the solution but it's  good to discussed together.